test/eni/IPRelease.go

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package eniTest
```

Comment:
We recently had a [PR](https://github.com/cilium/cilium/pull/18621) that changed all the uppercase file names to lowercase, so let's rename this file to lowercase. Might make sense to also use a lowercase package name too